vendor/engines/secure_rooms/config/routes.rb

```diff
@@ -5,4 +5,8 @@
       resources :price_policies, controller: "secure_room_price_policies", except: :show
     end
   end
+
+  namespace :secure_rooms_api do
+    post '/scan', to: 'base#scan'
```

Comment:
Would it make sense to have a `ScansController#create` and post to that? If we just have the one endpoint, it might not be worth the extra controller, but it currently feels weird with `BaseController`. Usually when I see something like that, I expect it to get subclassed.